components/link/test/link.visual-diff.js

```diff
@@ -20,35 +20,45 @@ describe('d2l-link', () => {
 
 	after(async() => await browser.close());
 
-	['print', 'screen'].forEach((mediaType) => {
+	[
+		{ name: 'wc-standard' },
+		{ name: 'wc-main' },
+		{ name: 'wc-small' },
+		{ name: 'wc-inline', selector: '#wc-inline d2l-link' },
+		{ name: 'wc-inline-paragraph', selector: '#wc-inline-paragraph d2l-link' },
+		{ name: 'wc-block' },
+		{ name: 'wc-clamp-one-line' },
+		{ name: 'wc-clamp-two-lines' },
```

Comment:
Sometimes I've included an "unbreakable" case for the overflow clamping scenarios. Might be worthwhile here as well.